chain/client/src/sync/state.rs

```diff
@@ -378,6 +397,39 @@ impl StateSync {
         }
         let split_states = epoch_manager.will_shard_layout_change(&prev_hash)?;
 
+        for msg in self.state_parts_mpsc_rx.try_iter() {
```

Comment:
Wouldn't this fit better in `sync_shards_download_parts_status()`?

Reply:
I am thinking of moving it in a different function and call that function from `StateSync::run()`.

Reply:
moved the logic to `process_downloaded_parts`